reference/ToDo/src/ToDo/Views/TaskListPage.xaml

```diff
@@ -113,7 +113,7 @@
                       <utu:AutoLayout PrimaryAxisAlignment="End"
                                       Orientation="Horizontal">
                         <ToggleButton utu:AutoLayout.CounterAlignment="Start"
-                                      IsChecked="{Binding IsImportant}"
+                                      IsChecked="{Binding IsImportant, Mode=TwoWay}"
```

Comment:
We shouldn't need this TwoWay binding, it should be properly coming downstream from the resulting update that is done as part of the ToggleIsImportant Command. It looks like we are subscribing to the right messages being sent by the ITaskService.

If you look [here](https://github.com/unoplatform/Uno.Samples/blob/d82132390ce1e59f9ab7d66a16fc5ac95651f455/reference/ToDo/src/ToDo/Business/TaskService.cs#L32) this is what is being called when we update a ToDoTask. You'll notice that we are sending a `EntityMessage<ToDoTask>`. So we should be observing the messenger for our `IListState<ToDoTask> Tasks` property in the view model. I see we are doing this and combining it with the Entity [here](https://github.com/unoplatform/Uno.Samples/blob/f97fccb502d3c890b72c94a46a6528742b9e5bb6/reference/ToDo/src/ToDo/Presentation/TaskListViewModel.cs#L28)

I'd be curious if this gets fixed if we were to remove the added TwoWay binding and add something like `messenger.Observe(Tasks, t => t.Id);` to the list of Observe calls in the constructor of our viewmodel.